Dockerfile

```diff
@@ -15,8 +15,7 @@ RUN mvn package
 
 ### STAGE 2: Setup ###
 
-FROM openjdk:8-jre-alpine
-
+FROM openjdk:12-jdk-oracle
 
```

Comment:
Is it better to generate a docker image with the same version or with the next one?

Reply:
As above, I had the same question when I saw the upgrade to openJDK11 on the seed-java and I understood that this is the way that you want to do regarding Docker images. So, what is your proposal for the openjdk? Do you want to fix a version? Which will be the tag for the next one?:

`FROM openjdk:xxxxxxx`

Thank you so much!

Reply:
It was a @vinyulis bad day!

Reply:
I do not agree with this last comment @alfonsserra :) . I think that in seed-jee Dockerfile the java version when wildfly is executed is the java version of jboss/wildfly:15.0.0.Final image. For this image the version is open-jdk11. I only put open-jdk12 for maven and I decided to use this one instead 11 because alpine is more lightweight than others.